devTools/Dockerfile-php70-xdebug

```diff
@@ -1,4 +1,7 @@
 FROM php:7.0-cli
 
 RUN pecl install xdebug && docker-php-ext-enable xdebug
+RUN apt-get -y update && apt-get install -y expect
+RUN useradd --shell /bin/bash infection
```

Comment:
am I right we need it to automatically use `bash` when we are on docker?

Reply:
We just follow the principle of the least surprise here, because people are used to bash, and the default shell in most Linux'es these days is `/bin/sh`, which is, well, quite ugly outside of scripts. That is to say there's no particular need to use `/bin/bash` as a shell here, just a matter of future convenience.